services/coolifyService.js

```diff
@@ -359,9 +387,10 @@ coolify.alphaNumericName = function(name) {
 		let cool_name_curr = name.allReplace(coolifyDicts.curr);
 		let cool_name_fairy = name.allReplace(coolifyDicts.fairy);
 		let cool_name_dot = name.allReplace(coolifyDicts.dot);
-    let cool_name_doublestruck = name.allReplace(coolifyDicts.doublestruck);
+    		let cool_name_doublestruck = name.allReplace(coolifyDicts.doublestruck);
```

Comment:
Fix the indentation.

Reply:
Fixed the indentation https://github.com/NJACKWinterOfCode/cool-name-api/pull/15/commits/5729fd65abecf22197242ac91504d633a7f9d7f7